src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,81 @@
+import './Pagination.css';
+import { IPaginationProps } from '../../types/interfaces';
+import { Context } from '../../context/context';
+import { useContext } from 'react';
+import { useSearchParams } from 'react-router-dom';
+
+export default function Pagination({
+  currentPage,
+  allPage,
+  linkNextPage,
+  linkPrevPage,
+}: IPaginationProps) {
+  const {
+    searchString,
+    setIsLoading,
+    setCards,
+    setCurrentPage,
+    setLinkNextPage,
+    setLinkPrevPage,
+  } = useContext(Context);
+  // eslint-disable-next-line
+  const [searchParams, setSearchParams] = useSearchParams();
```

Comment:
можно так написать и не дизейблить

```suggestion
  const [, setSearchParams] = useSearchParams();
```